packages/create-app/templates/default-app/packages/backend/package.json.hbs

```diff
@@ -41,7 +41,7 @@
     "@backstage/plugin-search-backend-node": "^{{version '@backstage/plugin-search-backend-node'}}",
     "@backstage/plugin-techdocs-backend": "^{{version '@backstage/plugin-techdocs-backend'}}",
     "app": "link:../app",
-    "better-sqlite3": "^9.0.0",
```

Comment:
Seen as this is in the `create-app` template, I think we should have separate changeset for `create-app` that tells the diff of what to do in `packages/backend/package.json`

Reply:
Thanks Ben, was going to update this today. 🚀